resources/views/layouts/user-guide.blade.php

```diff
@@ -0,0 +1,125 @@
+<!doctype html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- CSRF Token -->
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+    <title>{{ config('app.name') }} | @yield('title')</title>
+
+    <!-- Fonts -->
+    <link rel="dns-prefetch" href="//fonts.bunny.net">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
```

Comment:
suggestion, to improve site performance if you can download the external css / reference the better, though this will work